orc8r/cloud/go/tools/swaggergen/generate/generate.go

```diff
@@ -242,6 +288,17 @@ func StripAndWriteSwaggerSpecs(specs map[string]MagmaSwaggerSpec) error {
 	return nil
 }
 
+// RetrieveServiceName parses the targetFilepath for its originating service.
+func RetrieveServiceName(targetFilePath string, rootDir string) (string, error) {
```

Comment:
This is only being used in one spot. Consider making it a private function called from GenerateStandaloneSpec